_upgrade-to/upgrade-to.md

```diff
@@ -144,7 +144,7 @@ If you are migrating an Open Distro for Elasticsearch cluster, we recommend firs
 
    1. Port your settings from `elasticsearch.yml` to `opensearch.yml`. Most settings use the same names. At a minimum, specify `cluster.name`, `node.name`, `discovery.seed_hosts`, and `cluster.initial_cluster_manager_nodes`.
 
-   1. (Optional) If you're actively connecting to the cluster with legacy clients that check for a particular version number, such as Logstash OSS, add a [compatibility setting]({{site.url}}{{site.baseurl}}/clients/agents-and-ingestion-tools/) to `opensearch.yml`:
+   1. (Optional) If you're actively connecting to the cluster with legacy clients that check for a particular version number, such as Logstash OSS, add a [compatibility setting]({{site.url}}{{site.baseurl}}/https://opensearch.org/docs/2.7/tools/index/#compatibility-matrices/) to `opensearch.yml`:
```

Comment:
This link doesn't work for me. I built local to try it. (But the 404 page works!)

Reply:
It works for me locally. @Naarcha-AWS or @kolchfa-aws - do you mind testing this link? Thank you!

Reply:
Remove the first prefix from the link (starting with https)